app/controllers/tenants/projects_controller.rb

```diff
@@ -64,7 +74,11 @@ def new_categories
   def new_params
     new_param = project_params
     new_param[:categories] = new_categories
-    new_param[:tenant_id] = current_user.tenant.id
+    if current_user.nil?
```

Comment:
I need to read this section of code regarding new_param... but I trust this is doing what it's supposed to be. Can I take a look at this with you?